src/api/artifact/abstractor/resolver/chart/chart.go

```diff
@@ -30,17 +32,18 @@ import (
 const (
 	// ArtifactTypeChart defines the artifact type for helm chart
 	ArtifactTypeChart        = "CHART"
-	AdditionTypeValues       = "VALUES.YAML"
-	AdditionTypeReadme       = "README"
+	AdditionTypeValues       = "values.yaml"
+	AdditionTypeReadme       = "README.md"
```

Comment:
It's better to use Upper case for const definitions

Reply:
I think [pascal case](http://wiki.c2.com/?PascalCase) here is ok.

Here is the const definitions in `distribution`:

```go
const (
	// MediaTypeManifest specifies the mediaType for the current version. Note
	// that for schema version 1, the the media is optionally "application/json".
	MediaTypeManifest = "application/vnd.docker.distribution.manifest.v1+json"
	// MediaTypeSignedManifest specifies the mediatype for current SignedManifest version
	MediaTypeSignedManifest = "application/vnd.docker.distribution.manifest.v1+prettyjws"
	// MediaTypeManifestLayer specifies the media type for manifest layers
	MediaTypeManifestLayer = "application/vnd.docker.container.image.rootfs.diff+x-gtar"
)
```

How terrible when we use these values if they are in upper case?

Reply:
I mean the enumerated const values. They are used as enumerated values.

@wy65701436 It doesn't work here. The addition types defined here are converted into lower case when populating the addition links: https://github.com/goharbor/harbor/blob/master/src/api/artifact/controller.go#L442.
And they are converted into upper case when calling the resolver.ResolveAddition: https://github.com/goharbor/harbor/blob/master/src/server/v2.0/handler/artifact.go#L186

Reply:
> I mean the enumerated const values. They are used as enumerated values.
> 

I got it.  +1

Reply:
use the upper in const values.